src/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py

```diff
@@ -341,6 +343,10 @@ def generate_config():
         #
         # Comments (beginning with #) do not need to be removed. Optional parameters left blank are treated as false.
         ##############################################################################################################
+        # Required: Disk size of largest sample in cohort. Must be set by user or an error is raised.
```

Comment:
"Disk size of sample" makes no sense because a sample neither is or has a disk. Do you mean the "largest amount of space occupied by the data for any one sample in the batch at any point in time during the workflow" or is this simply the "size of the largest sample in the batch"?